packages/strapi-design-system/src/Link/Link.js

```diff
@@ -43,7 +43,7 @@ const IconWrapper = styled(Box)`
   display: flex;
 `;
 
-export const Link = ({ href, to, children, disabled, startIcon, endIcon, ...props }) => {
+export const Link = ({ href, to, children, disabled, startIcon, endIcon, isExternal, ...props }) => {
```

Comment:
Some comments on `Link`:
- Currently, it's not possible to tab to the `Link` in `SimpleMenu` because the ref isn't passed which needs to be fixed in SimpleMenu v1/v2 components, for Link v1 you will need to accept a `ref` with `forwardRef` and pass it to `LinkWrapper`
- I think we should remove `isExternal` here as we don't need it, this component use `react-router` so we only need to check on to or `href` prop `to` know if we are in an internal or external context. Plus it would be a breaking change since all `Link` used for external links without this prop in Strapi would lose their icon. Can you remove `isExternal` from prop and fix the condition for icon appearance below please?
```js
{href && (
   <IconWrapper as="span" aria-hidden={true} paddingLeft={2}>
       <ExternalLink />
   </IconWrapper>
)}
```
- the style of an external `Link` is broken when the element is focused in `SimpleMenu` because of `buttonFocusStyle` which I think we don't need here, we can rely on `ThemeProvider` `*:focus-visible` style and add the border to `LinkWrapper` instead of `buttonFocusStyle`
```js
const LinkWrapper = styled.a`
  ...
  ${buttonFocusStyle}; <== to remove

  &:focus-visible {
    border-radius: ${({ theme }) => theme.borderRadius};
  } <=== to add 
`;
```